### DISCUSSION TITLE
Current changes in Mongodb collection is reflected on localhost but not on deployed version in vercel

### DISCUSSION THREAD

**Discussion by ShubhradipSarkar** [vercel/next.js#61884] [Help]
### Summary

I am using axios to get data from a collection.
If I go to the collection and change/edit some document, then reload the localhost, in UI the change is reflected, but in vercel hosted application, it doesn't reflect the changes
Then if I do any minor change in code like a space or something and then push in GitHub, the deployment happens and then the hosted one reflects the changes
Any help will be much appreciated 👍

**Comment by arvindgt** [comment]
I was facing the same problem with my app and the below thing worked for me. You can give a try with adding the 'revalidate' in your routes or api file (wherever you are connecting to database and trying to fetch the data).

**export const revalidate = 0;**

One sample of my code is below:

`import connectMongoDB from "../../../lib/mongodb";
import User from "../../../../models/user";
import { NextResponse } from "next/server";

export const revalidate = 0; // this is the new line added

export async function GET()  {
    await connectMongoDB();
    const users = await User.find();
    return NextResponse.json({users});
};`
